lib/active_model/serializer/association.rb

```diff
@@ -10,24 +10,86 @@ class Serializer
     #  Association.new(:comments, { serializer: CommentSummarySerializer })
     #
     class Association < Field
+      # TECHDEBT: remove unneeded accessors
```

Comment:
Move unneeded accessors below `private`, so we know they're not meant to be used publicly.